pytest-selenium/tests/test_highlighting_56.py

```diff
@@ -2,16 +2,19 @@
 
 import pytest
 from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
 
 from pages.accounts import Signup
 from tests import markers
 from pages.osweb import WebBase
 from pages.content import Content
 from utils.utility import Highlight, Utilities
+from utils import utility
```

Comment:
instead of importing the whole utility file (this imports two classes then imports those classes again plus anything else in `utility`) just pull in the new function:
`from utils.utility import Highlight, Utilities, get_search_term`
(the function goes at the end because, when sorting, lowercase letters have a larger character value than uppercase ones)